app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -129,10 +137,38 @@ export const FixedSessionsMapCtrl = (
         params.update({data: {usernames: profiles.join(", ")}});
         $scope.sessions.fetch();
       });
+
+      setTimeRangeFilter(params, elmApp)
+
+      elmApp.ports.updateTimeRange.subscribe((timeRange) => {
+        params.update({data: {timeFrom: timeRange.timeFrom, timeTo: timeRange.timeTo}});
+        $scope.sessions.fetch();
+      });
     });
   }
 }
 
+const setTimeRangeFilter = (params, elmApp) => {
+  if (document.getElementById("daterange")) {
+    $('input[id="daterange"]').daterangepicker({
```

Comment:
Have you considered using the same selector in the if and here? I guess the id could be enough